.github/workflows/ubuntu.yml

```diff
@@ -8,7 +8,12 @@ on:
 
 jobs:
   test-linux:
-    runs-on: ubuntu-latest
+    name: ${{ matrix.os }} Headless
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-18.04, ubuntu-20.04]
```

Comment:
I'm guessing we can merge this with other ymls and use matrix across all OSes. But that can be a different change.

Reply:
done